lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -55,7 +55,7 @@ def investigate(processed_source)
 
         private
 
-        KEYWORDS = %w[begin class def end module].freeze
+        KEYWORDS = ['begin ', 'class ', 'def ', 'end ', 'module '].freeze
```

Comment:
I think it'd be better to adjust the regex. I find it odd every time I see some array with trailing whitespaces.

Reply:
There's no regex check for this one. However, `KEYWORDS` can become a word array again by doing this:

```ruby
line.start_with?(*KEYWORDS.map { |word| word + ' ' })
```

Reply:
Ah... this seems better:

```ruby
KEYWORDS.any? { |word| line =~ /^#{word}\s+/ }
```